source/glossary.rst

```diff
@@ -93,6 +93,22 @@ The Operator options glossary
 
       :ref:`pmm.image<pmm-image>`
 
+   imagePullPolicy
+
+     ..
+
+      :ref:`backup.imagePullPolicy<backup-imagepullpolicy>`
+
+      :ref:`pgBadger.imagePullPolicy<pgbadger-imagepullpolicy>`
+
+      :ref:`pgBouncer.imagePullPolicy<pgbouncer-imagepullpolicy>`
+
+      :ref:`pgPrimary.imagePullPolicy<pgprimary-imagepullpolicy>`
+
+      :ref:`pgReplicas.REPLICA_NAME.imagePullPolicy<pgreplicas-imagepullpolicy>`
```

Comment:
We will not have separate option for `pgReplicas`. The `imagePullPolicy` from `pgPrimary` will be used for `pgReplicas`.

Reply:
@fiowro it was fixed and now we can merge it. So, please fix conflicts.